src/styles/components/_audiovideo.scss

```diff
@@ -156,10 +194,20 @@
     background: $video-background;
     display: block;
     max-height: 100%;
+    width: 100%;
```

Comment:
Properties should be ordered background, display, max-height, opacity, transition-duration, transition-property, width